api/policy.go

```diff
@@ -56,45 +57,61 @@ This would prevent someone from toggling something to disabled or 0 respectively
-	AlertEnabled  *bool  `json:"alertEnabled,omitempty" yaml:"alertEnabled,omitempty"`
-	AlertProfile  string `json:"alertProfile,omitempty" yaml:"alertProfile,omitempty"`
+	EvaluatorID   string   `json:"evaluatorId,omitempty" yaml:"evaluatorId,omitempty"`
+	PolicyID      string   `json:"policyId,omitempty" yaml:"policyId,omitempty"`
+	PolicyType    string   `json:"policyType,omitempty" yaml:"policyType,omitempty"`
+	QueryID       string   `json:"queryId,omitempty" yaml:"queryId,omitempty"`
+	Title         string   `json:"title,omitempty" yaml:"title,omitempty"`
+	Enabled       *bool    `json:"enabled,omitempty" yaml:"enabled,omitempty"`
+	Description   string   `json:"description,omitempty" yaml:"description,omitempty"`
+	Remediation   string   `json:"remediation,omitempty" yaml:"remediation,omitempty"`
+	Severity      string   `json:"severity,omitempty" yaml:"severity,omitempty"`
+	Limit         *int     `json:"limit,omitempty" yaml:"limit,omitempty"`
+	EvalFrequency string   `json:"evalFrequency,omitempty" yaml:"evalFrequency,omitempty"`
+	AlertEnabled  *bool    `json:"alertEnabled,omitempty" yaml:"alertEnabled,omitempty"`
+	AlertProfile  string   `json:"alertProfile,omitempty" yaml:"alertProfile,omitempty"`
+	Tags          []string `json:"tags,omitempty" yaml:"tags,omitempty"`
 }
 
 type Policy struct {
-	EvaluatorID    string `json:"evaluatorId"`
-	PolicyID       string `json:"policyId"`
-	PolicyType     string `json:"policyType"`
-	QueryID        string `json:"queryId"`
-	Title          string `json:"title"`
-	Enabled        bool   `json:"enabled"`
-	Description    string `json:"description"`
-	Remediation    string `json:"remediation"`
-	Severity       string `json:"severity"`
-	Limit          int    `json:"limit"`
-	EvalFrequency  string `json:"evalFrequency"`
-	AlertEnabled   bool   `json:"alertEnabled"`
-	AlertProfile   string `json:"alertProfile"`
-	Owner          string `json:"owner"`
-	LastUpdateTime string `json:"lastUpdateTime"`
-	LastUpdateUser string `json:"lastUpdateUser"`
+	EvaluatorID    string   `json:"evaluatorId"`
+	PolicyID       string   `json:"policyId"`
+	PolicyType     string   `json:"policyType"`
+	QueryID        string   `json:"queryId"`
+	Title          string   `json:"title"`
+	Enabled        bool     `json:"enabled"`
+	Description    string   `json:"description"`
+	Remediation    string   `json:"remediation"`
+	Severity       string   `json:"severity"`
+	Limit          int      `json:"limit"`
+	EvalFrequency  string   `json:"evalFrequency"`
+	AlertEnabled   bool     `json:"alertEnabled"`
+	AlertProfile   string   `json:"alertProfile"`
+	Tags           []string `json:"tags"`
+	Owner          string   `json:"owner"`
+	LastUpdateTime string   `json:"lastUpdateTime"`
+	LastUpdateUser string   `json:"lastUpdateUser"`
+}
+
+func (p *Policy) HasTag(t string) bool {
+	for _, v := range p.Tags {
+		if t == v {
+			return true
+		}
+	}
+	return false
```

Comment:
Could this be condensed```suggestion
			return  t == v
	}
```